p2p/net/mock/mock_stream.go

```diff
@@ -268,20 +277,19 @@ func (s *stream) transport() {
 			return
 		case o := <-s.toDeliver:
 			if err := deliverOrWait(o); err != nil {
-				s.resetWith(err)
+				s.cancelWrite(err)
 				return
 			}
 		case <-timer.C: // ok, due to write it out.
 			if err := drainBuf(); err != nil {
-				s.resetWith(err)
+				s.cancelWrite(err)
 				return
 			}
 		}
 	}
 }
 
-func (s *stream) resetWith(err error) {
+func (s *stream) cancelWrite(err error) {
```

Comment:
Are there any test changes downstream that would be effected by this change?

Reply:
I believe I needed this to fix an issue in the bitswap tests.